src/measures/finite.jl

```diff
@@ -113,6 +113,46 @@ function call(m::BACC, ŷm, ym, wm::Union{Nothing,Arr{<:Real}}=nothing)
     return score
 end
 
+# ---------------------------------------------------
+# kappa
+
+struct Kappa <: Aggregated end
+
+metadata_measure(Kappa;
+                 instances  = ["kappa"],
+                 target_scitype           = FiniteArrMissing,
+                 prediction_type          = :deterministic,
+                 orientation              = :score,
+                 supports_weights         = false)
+
+@create_aliases Kappa
+
+@create_docs(Kappa,
+body=
+"""
+A metric to measure agreement between predicted labels and the ground truth. 
+See [https://en.wikipedia.org/wiki/Cohen%27s_kappa](https://en.wikipedia.org/wiki/Cohen%27s_kappa)
+
+$INVARIANT_LABEL
+""",
+scitype=DOC_FINITE)
+
+# calling behaviour:
+function (::Kappa)(cm::ConfusionMatrixObject{C}) where C
+    # relative observed agreement - same as accuracy
+    p₀ = sum(diag(cm.mat))/sum(cm.mat)
+
+    # probability of agreement due to chance - for each class cᵢ, this would be: (#predicted=cᵢ)/(#instances) x (#observed=cᵢ)/(#instances)
+    pₑ = sum(sum(cm[j, :]) * sum(cm[:, j]) for j in 1:C)/sum(cm.mat)^2
```

Comment:
The following should be more efficient, even in the case where C>>2. This take advantage of the fact that `ConfusionMatrixObject` wraps a square matrix of type `Matrix{Int}`.
```suggestion
    rows_sum = sum!(similar(cm.mat, 1, C), cm.mat) # 1 x C matrix
    cols_sum = sum!(similar(cm.mat, C, 1), cm.mat) # C X 1 matrix
    pₑ = first(rows_sum*cols_sum)/sum(rows_sum)^2
```

Reply:
Makes sense. How about we have something like this (a little more simpler) ?

```
rows_sum = sum(cm.mat, dims=2)
cols_sum  = sum(cm.mat, dims=1)
pₑ = first(cols_sum*rows_sum)/sum(rows_sum)^2
```

Reply:
> rows_sum = sum(cm.mat, dims=2)
> cols_sum  = sum(cm.mat, dims=1)
> pₑ = first(cols_sum*rows_sum)/sum(rows_sum)^2

Yeah, that also works, but the only issue here is that `rows_sum = sum(cm.mat, dims=2)` isn't as efficient as the original implementation. But as the C grows in size this is still quite efficient. You may use this option if you want.
It's up to you.